dm/loader/lightning.go

```diff
@@ -340,7 +341,7 @@ func GetLightningConfig(globalCfg *lcfg.GlobalConfig, subtaskCfg *config.SubTask
 		}
 		cfg.Checkpoint.DSN = cpPath
 	}
-	cfg.Checkpoint.KeepAfterSuccess = lcfg.CheckpointOrigin
+	cfg.Checkpoint.KeepAfterSuccess = lcfg.CheckpointRemove
```

Comment:
PTAL~ 
Pls help review whether it's ok to remove the lightning checkpoint after the import.
@GMHDBJD @lance6716

Reply:
The problem is if the worker process crashes between line 430 and line 432, after failover data will be imported twice. The candidate actions can be
- keep CheckpointOrigin, but user may complain that a strange database is left in thier cluster
- change to CheckpointRemove and ignore the risk
- add a callback function to lightning, update the DM load unit checkpoint before removing lightning's checkpoint
- refactor lightning checkpoint schema, put all tables under DM task meta database.

I'm neutral about them

Reply:
Already add the 'todo' and wait for another pr to fix it.
May be 'way 3' is the easier way.